test/integration/com/unifina/service/CommunitySecretServiceIntegrationSpec.groovy

```diff
@@ -116,6 +118,59 @@ class CommunitySecretServiceIntegrationSpec extends Specification {
 		result.communityAddress == communityAddress
 	}
 
+	void "update() only secret"() {
+		setup:
+		CommunitySecret s1 = new CommunitySecret(
+			name: "secret 1",
+			secret: "secret#1",
+			communityAddress: communityAddress,
+		)
+		s1.save(validate: true, failOnError: true)
+		CommunitySecretCommand cmd = new CommunitySecretCommand(
+			secret: "new secret",
```

Comment:
Could we drop the secret field from CommunitySecretCommand?

Reply:
that would make it like apikey, sure. Then you couldn't input your own secret, I guess? I think I'll keep it like it is for now